doc/tutorial/Client.lhs

```diff
@@ -107,6 +107,7 @@ data BaseUrl = BaseUrl
   { baseUrlScheme :: Scheme -- ^ URI scheme to use
   , baseUrlHost :: String   -- ^ host (eg "haskell.org")
   , baseUrlPort :: Int      -- ^ port (eg 80)
+  , baseUrlPath :: String   -- ^ path (eg "/a/b/c")
```

Comment:
But I suppose you can't turn this into an not ignored block. Because then the subsequent code couldn't disambiguate this `BaseUrl` from the (correct) imported `BaseUrl`. No?